PidginMacOSIntegration.xcodeproj/project.pbxproj

```diff
@@ -35,6 +35,7 @@
 /* End PBXCopyFilesBuildPhase section */
 
 /* Begin PBXFileReference section */
+		9DFC539927488134005F0927 /* Dependencies.xcconfig */ = {isa = PBXFileReference; lastKnownFileType = text.xcconfig; path = Dependencies.xcconfig; sourceTree = "<group>"; };
```

Comment:
Some kind of a `Run Script` phase or similar config is missing - currently it doesn't run the script at all and that would be the expected behavior (and commits mention it).

After manually running the script with correct env vars (I can only hope that xcode sets correct project dir as of now) it builds correctly through both XCode and terminal command.